Sources/FoundationNetworking/URLSession/URLSessionConfiguration.swift

```diff
@@ -216,14 +216,9 @@ open class URLSessionConfiguration : NSObject, NSCopying {
      Note that these headers are added to the request only if not already present. */
     open var httpAdditionalHeaders: [AnyHashable : Any]? = nil
     
-    #if NS_CURL_MISSING_MAX_HOST_CONNECTIONS
     /* The maximum number of simultaneous persistent connections per host */
-    @available(*, deprecated, message: "This platform doles not support selecting the maximum number of simultaneous persistent connections per host. This property is ignored.")
```

Comment:
We lose this part, but that's the consequence until swiftpm supports something like cmake does here.